examples/pytorch_sparse_quantized_transfer_learning/README.md

```diff
@@ -0,0 +1,94 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# PyTorch Sparse-Quantized Transfer Learning with SparseML
+
+[Pruning](https://neuralmagic.com/blog/pruning-overview/) and
+[quantization](https://arxiv.org/abs/1609.07061) are well-established methods for accelerating
+neural networks.  Individually, both methods yield significant speedups for CPU inference
+(a theoretical maximum of 4x for INT8 quantization) and can make CPU deployments an attractive
+option for real-time model inference.
+
+Sparse-quantized models leverage both techniques and can achieve speedups upwards of 6-7x when using
```

Comment:
can we get a link out for the 6-7x to the resnet50 blog? Also, can we change to just 7x or ~7x?